idle-application/src/main/kotlin/com/swm/idle/application/chat/domain/ChatMessageService.kt

```
@@ -0,0 +1,24 @@
+    fun createByUser(roomId: UUID, userId: UUID, contents: List<ChatMessage.Content>): ChatMessage {
+        return ChatMessage(
+            roomId = roomId,
+            senderId = userId,
+            senderType = SenderType.USER,
+            contents = contents
+        )
+    }
+
+    fun publish(it: ChatMessage) {
+
+    }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)